apps/example/src/entrypoints/reactions.ts

```diff
@@ -58,7 +58,7 @@ function createSource() {
         query: QueryOptions.create({}),
       })
       const checkpoints = DynamoCheckpoints.create(
-        config.context,
+        context,
         config.cache,
         300, // 5 minutes
```

Comment:
5 minutes what

Reply:
also assign categories to a `handledCategories` to show that it needs to align
and pass groupName as an arg to createSource
for ddb, move mode toward front of arg list and handler to the end to match order in which things are relevant